app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Contestants</h1>
+
+<% @contestants.each do |contestant| %>
+<ul>
+  <li><%= contestant.name %> <br>
+  <%= "Projects: " %>
+    <% contestant.projects.each do |project| %>
+      <ul>
+        <li><%= "#{project.name}" %></li>
```

Comment:
since you're already in erb tags, you can just call <%= project.name %> since the return value of this is a string already... no need to interpolate :)